citylearn/electric_vehicle.py

```diff
@@ -0,0 +1,493 @@
+import inspect
```

Comment:
Please, perform general code checks like:

1. Variables being spelled in full
2. Docstrings for all methods except `setters`
3. Type hints for all method parameters and returned value.